interface-definitions/ssh.xml.in

```diff
@@ -206,6 +206,39 @@
             </properties>
             <defaultValue>22</defaultValue>
           </leafNode>
+          <node name="rekey-limit">
+            <properties>
+              <help>SSH session rekey limit</help>
+            </properties>
+            <children>
+              <leafNode name="data">
+                <properties>
+                  <help>Threshold data in Megabytes</help>
+                  <valueHelp>
+                    <format>u32:1-65535</format>
+                    <description>Megabytes</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-65535"/>
+                  </constraint>
+                </properties>
+                <defaultValue>default</defaultValue>
```

Comment:
If you set defaultValue to default, thete should be a completionHelp, valueHelp and constraint, too

Reply:
Please also provide a `valueHelp` for `default`

```xml
                  <valueHelp>
                    <format>default</format>
                    <description>foo bar</description>
                  </valueHelp>
```